simulation_parameters/Constants.cs

```diff
@@ -657,6 +658,8 @@ public static class Constants
     /// </summary>
     public const string SPECIES_NAME_REGEX = "^(?<genus>[^ ]+) (?<epithet>[^ ]+)$";
 
+    public const string SPECIES_NAME_DICT = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890 ";
+
```

Comment:
```suggestion
    public const string SPECIES_NAME_REGEX = "^(?<genus>[a-zA-Z0-9]+) (?<epithet>[a-zA-Z0-9]+)$";
```
you can merge these two checks into one